libnetwork/drivers/bridge/port_mapping_linux.go

```diff
@@ -802,7 +802,7 @@ func setPerPortForwarding(b portBinding, ipv iptables.IPVersion, bridgeName stri
 		"-j", "ACCEPT",
 	}
 	rule := iptRule{ipv: ipv, table: iptables.Filter, chain: DockerChain, args: args}
-	if err := appendOrDelChainRule(rule, "MASQUERADE", enable); err != nil {
+	if err := appendOrDelChainRule(rule, "OPEN PORT", enable); err != nil {
```

Comment:
Why change?

Reply:
These functions are horrible; from the "before" code, I really assumed `MASQUERADE` was some special value required for the code to  work (wondered "can we just use _two words_ here?". But ... all it is is a custom message to include in an error that's produced. It's just an abstraction to not have to write an error message here 🤦 
 https://github.com/moby/moby/blob/8cc4029162401663ae10be601ffb69d9cf2dfb46/libnetwork/drivers/bridge/setup_ip_tables_linux.go#L336

Reply:
I think I actually may have a branch somewhere that changed these. But can't find them 😂